### PR TITLE
[#121272913] Fixed grid-gutter-width value & Update bootstrap version to 3.3.6

### PR DESCRIPTION
**Pull request by toshipon**
I fixed following things:
- Fixed grid-gutter-width value to fix grid-colums value changed (see attached images how it works)
- Fixed 2x image file name rule (the issue causes [this](http://gengo.github.io/style-guide/#apple-touch-icons))
- Fixed csslint error
- Added script commands into package.json

change logs for bootstrap [v3.3.4](https://github.com/twbs/bootstrap/releases/tag/v3.3.4),  [v3.3.5](https://github.com/twbs/bootstrap/releases/tag/v3.3.5),  [v3.3.6](https://github.com/twbs/bootstrap/releases/tag/v3.3.6)

### Before 
inside of `form-horizontal` class, `margin-left` and `margin-right` are used original bootstrap values

screenshot 2016-06-21 17 40 51

### After
screenshot 2016-06-21 17 41 14

### CODE REVIEW

**Comment by hayate** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -33,5 +33,9 @@
   "repository": {
     "type": "git",
     "url": "git@github.com:gengo/style-guide.git"
+  },
+  "scripts": {
+    "dev": "node_modules/grunt-cli/bin/grunt dev",
```
Comment (patatepartie) [inline package.json]: You don't need to prefix by the path. All the bins in `node_modules` are automatically added to `scritpts` path.
So: `"dev": "grunt dev"`

Comment (toshipon) [inline package.json]: oh you're right . i didn't know that .. fixed it 🙇

**Comment by patatepartie** [comment]
LGTM

**Pull request merged**